src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp

```diff
@@ -788,6 +790,7 @@ static void topPrintUsage(const xcldev::device *dev, xclDeviceUsage& devstat)
 
     dev->m_cu_usage_stringize_dynamics(lines);
 
+    dev->clearSensorTree();
```

Comment:
Why do we need to clear the sensor tree? Doesn't it gets overwritten?

Reply:
it didn't, instead it will have duplicate fields if you add the same entries.

and sensor_tree free when xbutil terminated. So we have to find a way to clear it.

Reply:
I see.